presto-main/src/main/java/com/facebook/presto/memory/ClusterMemoryManager.java

```diff
@@ -417,6 +417,7 @@ private synchronized MemoryPoolAssignmentsRequest updateAssignments(Iterable<Que
                     }
                 }
                 if (biggestQuery != null) {
+                    log.warn("Moving query %s to the reserved pool", biggestQuery.getQueryId());
```

Comment:
why not info?

Reply:
Because it's generally dire situation.. unless you configured your cluster to have ~0 general pool as in https://github.com/prestodb/presto/issues/11239#issuecomment-421511717 .  I will change to `.info`. Thanks!

Reply:
I know this is already merged, but when there is only one variable part of a message, it's best to put that at end, to make it easier to search for the message:
```java
"Moving query to the reserved pool: %s"

Reply:
Same logic applies to error messages. Sometimes you need to restructure them a bit so that it's clear what the variable part means (when it'd be more clear in the middle).

Reply:
I agree. Fortunately in this case, if you see a text `Moving query <query-id> to the reserved pool`, you know which parts are variable and "moving query" prefix along is (currently) surprisingly unique :)